index.php

```diff
@@ -2,9 +2,46 @@
 transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" lang="de">
 <head profile="http://gmpg.org/xfn/11">
-  
-  
+ 
+<link rel="stylesheet" type="text/css" href="<?php bloginfo('stylesheet_directory'); ?>/resmio-api/css/resmio-style.css">
```

Comment:
@RNDWSR change it to get_stylesheet_directory_uri as described in pull request #19